system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/cells/microservices/MicroService.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Details from "./Details";
+import K8sObjects from "./K8sObjects";
+import Metrics from "./Metrics";
+import Paper from "@material-ui/core/Paper";
+import PropTypes from "prop-types";
+import React from "react";
+import Tab from "@material-ui/core/Tab";
+import Tabs from "@material-ui/core/Tabs";
+import TopToolbar from "../../common/TopToolbar";
+import {withStyles} from "@material-ui/core/styles";
+
+
```

Comment:
Double newlines. Keep single newline.

Reply:
Fixed in https://github.com/wso2/product-vick/blob/master/system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/components/cells/component/index.js